lib/xapi/file_bundle.rb

```diff
@@ -0,0 +1,28 @@
+require 'zip'
+module Xapi
+  # Module to return all the files in given path and zip them
+  class FileBundle
+    attr_reader :dir, :ignore_patterns
+    def initialize(dir, ignore_patterns=[])
+      @dir = dir
+      @ignore_patterns = ignore_patterns
+    end
+
+    def create_zip
```

Comment:
I avoid names that say what a method does, and try to name things for what the method returns. This is because the implementation could change (we could decide to create the zip elsewhere, and this method would just return the zip that we had created).